README.md

```diff
@@ -38,9 +49,12 @@ In an AJAX POST request, what does the argument of the `.done()` callback - in t
   .done(function(data) {
     console.log("success!");
   });
+
+-The argument 'data' represents the user info (name, about, and email) that is collected from user input if the post request was successful.
```

Comment:
Close - as Tom pointed out, it's actually a response from the server (not the actual user input), though by convention it's the object that was just POSTed.